packages/@aws-cdk/aws-apigateway/test/base-path-mapping.test.ts

```diff
@@ -65,7 +65,31 @@ describe('BasePathMapping', () => {
     });
 
     // WHEN
-    const invalidBasePath = '/invalid-/base-path';
```

Comment:
Please ensure that this string is still tested in the unit tests

Reply:
The above string is suitable for testing invalid base path mapping as slash is allowed after my changes. 
So instead I have created a new string for testing invalid character as shown below - 
https://github.com/VarunWachaspati/aws-cdk/blob/main/packages/%40aws-cdk/aws-apigateway/test/base-path-mapping.test.ts#L92